src/utilities/findPreviousSquad.js

```diff
@@ -0,0 +1,15 @@
+import store from '../Redux/store';
+
+function findPreviousSquad(roleIndex, rowIndex) {
```

Comment:
Should probably bundle these utilities into less files. Seems like a lot of utility files when there is common overlap to bundle them under a file name.